app/models/manageiq/providers/hawkular/datawarehouse_manager.rb

```diff
@@ -11,6 +11,11 @@ class Hawkular::DatawarehouseManager < ManageIQ::Providers::DatawarehouseManager
     DEFAULT_PORT = 80
     default_value_for :port, DEFAULT_PORT
 
+    CLIENT_KLASSES = {
```

Comment:
This forces the constants to be autoloaded before the class itself is loaded, so I'd advise against it. The later we manage to move constant loading the better.